### PR TITLE
Ignore IE11's and Edge's service handlers (close #379)

### LINKED ISSUE #379: Error occurs in _getEventListenerWrapper method in IE11
Added listener doesn't have 'call' method after removing related element

### PR DESCRIPTION
**Pull request by churkin**
\cc @inikulin @LavrovArtem

### CODE REVIEW

File: src/client/sandbox/event/listeners.js [hunk 1]
```diff
@@ -54,16 +54,20 @@ export default class Listeners extends EventEmitter {
...
+                        return listener.handleEvent.call(listener, e);
 
-            if (isObjectEventListener(listener))
-                return listener.handleEvent.call(listener, e);
+                    return listener.call(this, e);
+                }
+            }
 
-            return listener.call(this, e);
+            return null;
```
Comment (inikulin) [inline src/client/sandbox/event/listeners.js]: I don't get it. It will not work with regular handlers?

Comment (inikulin) [inline src/client/sandbox/event/listeners.js]: Got it now.

File: src/client/sandbox/event/listeners.js [hunk 2]
```diff
@@ -54,16 +54,20 @@ export default class Listeners extends EventEmitter {
 
     static _getEventListenerWrapper (eventCtx, listener) {
         return function (e) {
-            if (eventCtx.cancelOuterHandlers)
-                return null;
+            // NOTE: Ignore IE11's and Edge's service handlers (GH-379)
+            if (listener.toString() !== '[object FunctionWrapper]') {
```
Comment (inikulin) [inline src/client/sandbox/event/listeners.js]: I'd rather use variable here for clarity:
```js
var isIEServiceHandler = listener.toString() === '[object FunctionWrapper]';

if (!isIEServiceHandler) {
 ...
```

or revert logic: 
```js
if (listener.toString() === '[object FunctionWrapper]')
    return;

...
```

**Comment by inikulin** [comment]
This requires regression test. \r-

**Comment by churkin** [comment]
FPR

**Comment by LavrovArtem** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Pull request merged**